src/Pages/ProductInfo/Reviews.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import './Reviews.scss';
+
+class Reviews extends React.Component {
+  render() {
+    return (
+      <div className="review-component">
```

Comment:
```suggestion
      <div className="reviews">
```
- 컴포넌트 이름과 동일하게 네이밍 하는 것이 위코드 컨벤션입니다~

Reply:
수정 완료했습니다!